ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue

```diff
@@ -100,25 +100,28 @@
             label="Email Address"
             v-model="submittingParty.emailAddress"
             :rules="emailRules"
+            validate-on-blur
           />
 
           <!-- Phone Number -->
           <label class="generic-label" for="submitting-party-phone">Phone Number</label>
           <v-row no-gutters>
             <v-col>
               <v-text-field
+                v-mask="'(###) ###-####'"
                 filled
                 id="submitting-party-phone"
                 class="pt-4 pr-3"
                 label="Phone Number"
-                hint="Example: (555) 555-5555"
-                persistent-hint
                 v-model="submittingParty.phoneNumber"
                 :rules="phoneRules"
-              />
+                validate-on-
+                blur
```

Comment:
weird spacing on this prop. I believe this could be 1 line :) `validate-on-blur`

Reply:
fixed